shared-libs/transitions/src/transitions/mark_for_outbound.js

```diff
@@ -0,0 +1,60 @@
+const vm = require('vm');
+
+const config = require('../config'),
+      db = require('../db'),
+      logger = require('../lib/logger');
+
+const CONFIGURED_PUSHES = 'outbound';
+
+const arrayinate = object => Object.keys(object).map(k => {
+  object[k].key = k;
+  return object[k];
+});
+
+const configuredFor = doc => {
+  const pushes = arrayinate(config.get(CONFIGURED_PUSHES) || {});
+
+  return pushes.filter(conf => {
+    return conf.relevant_to && vm.runInNewContext(conf.relevant_to, {doc: doc});
```

Comment:
Maybe add a unit test to capture this behavior as expected.

Reply:
Great question!

It's unclear what we should do if a filter function of a transition can fail. I think the right and probably most consistent approach is to be even less careful than I was and just let it fail up to the caller wholesale. Then the framework can decide how to manage that (which for now looks similar to this code, but if we're going to handle it better it should be at this level)

Reply:
Actually because it doesn't really solve anything to have the `filter` function execute the relevant_to queries I changed the filter to just always work when there is config at all (which is more aligned with other transitions) and then moved all the action into the `onChange` function, which is properly wrapped by the framework to manage rejected promises etc.